addons/mail/static/src/core/public_web/discuss_sidebar.xml

```diff
@@ -1,25 +1,10 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <templates xml:space="preserve">
     <t t-name="mail.DiscussSidebar">
-        <div class="o-mail-DiscussSidebar d-flex d-print-none flex-column overflow-auto flex-shrink-0 h-100 border-end border-secondary z-1 o-mail-discussSidebarBgColor gap-1" t-att-class="{ 'o-compact': store.discuss.isSidebarCompact }">
-            <div class="o-mail-DiscussSidebar-top position-sticky justify-content-center top-0 z-2 o-mail-discussSidebarBgColor" t-att-class="{ 'pt-2 pb-1': !store.inPublicPage, 'pt-1': store.inPublicPage }">
-                <div class="d-flex align-items-center justify-content-center" t-att-class="{ 'flex-column gap-2': store.discuss.isSidebarCompact }">
-                    <t name="options-btn">
-                        <Dropdown position="'right-start'" menuClass="'o-mail-DiscussSidebar-floatingMenu bg-view border px-0 py-1 mx-2 my-0 min-w-0 shadow-sm'">
-                            <button class="o-mail-DiscussSidebar-optionsBtn btn btn-light p-0 align-items-center justify-content-center smaller border border-dark rounded-circle opacity-50 opacity-100-hover" title="Options" t-att-class="{ 'ms-auto': !store.discuss.isSidebarCompact, 'position-absolute': !store.discuss.isSidebarCompact and !store.inPublicPage, 'o-compact': store.discuss.isSidebarCompact }"><i class="fa fa-ellipsis-h fa-fw fa-lg" style="font-size: 15px;"/></button>
-                            <t t-set-slot="content">
-                                <div>
-                                    <DropdownItem class="'px-2 py-1 d-flex align-items-center rounded-0'" onSelected="() => this.store.discuss.isSidebarCompact = !this.store.discuss.isSidebarCompact">
-                                         <i class="fa fa-fw" t-att-class="{ 'fa-expand': store.discuss.isSidebarCompact, 'fa-compress': !store.discuss.isSidebarCompact }"/>
-                                         <span class="mx-2">
-                                             <t t-if="store.discuss.isSidebarCompact">Expand panel</t>
-                                             <t t-else="">Collapse panel</t>
-                                         </span>
-                                     </DropdownItem>
-                                </div>
-                            </t>
-                        </Dropdown>
-                    </t>
+        <div class="o-mail-DiscussSidebar d-flex d-print-none flex-column overflow-auto flex-shrink-0 h-100 border-end border-secondary z-1 o-mail-discussSidebarBgColor o-gap-0_5">
+            <div class="o-mail-DiscussSidebar-top position-sticky justify-content-center top-0 z-2 o-mail-discussSidebarBgColor" t-att-class="{ 'pt-2 pb-1': !store.inPublicPage, 'pt-1': store.inPublicPage }" t-ref="header-top">
+                <div class="d-flex align-items-center justify-content-center">
+                    <t name="startMeeting" />
                 </div>
             </div>
```

Comment:
IMO there is no reason to keep this in `public_web` now as there is nothing to display in the Topbar for public users.